iam.tf

```diff
@@ -15,6 +19,7 @@ resource "aws_iam_role" "claimant_api_kafka_consumer" {
 }
 
 resource "aws_iam_role_policy" "claimant_api_kafka_consumer" {
+  name   = "claimant_api_kafka_consumer"
```

Comment:
```suggestion
  name   = var.friendly_name
```

Reply:
Policy name aligns with the policy rather than the entity that uses the IAM role that has the policy attached - the names coincide to be the same in this case though.